README.md

```diff
@@ -1,2 +1,50 @@
 # terraform-examples
-Deploy Every Possible Flagsmith Adventure Terraform
+Terraform scripts for example deployments to popular cloud providers.
+
+## EKS
```

Comment:
Possibly one for a future PR, but I think it would possibly be better here to just have a list of the examples and link to their dirs, each of which have their own readme.

Reply:
Yes, I was going to do this as soon as we add an example for a second provider.